src/disk_io_thread.cpp

```diff
@@ -691,7 +696,7 @@ namespace libtorrent
 
 #if DEBUG_DISK_THREAD
 		DLOG("iovec_flushed: piece: %d block_offset: %d [ "
-			, int(pe->piece), block_offset);
+			, pe->piece.value(), block_offset);
```

Comment:
I see, this line doesn't get built normally, that's why this mistake went unnoticed